src/routes/admin/events/+page.svelte

```diff
@@ -0,0 +1,175 @@
+<script type="ts">
+	import { Button } from '@emerald-dao/component-library';
+	import Icon from '@iconify/svelte';
+	import { fly } from 'svelte/transition';
+	import type { Event } from '$lib/types/event/event.interface';
+	import { EventTypeEnum } from '$lib/types/content/metadata/event-types.enum';
+	import FloatTicket from '$lib/components/floats/FloatTicket.svelte';
+	import { generatedNft } from '$lib/features/event-generator/stores/EventGeneratorData';
+	import { createSearchStore, searchHandler } from '$stores/searchBar';
+	import { onDestroy } from 'svelte';
+	import EventSmallTicket from '$lib/components/events/EventSmallTicket.svelte';
+	import EventTicket from '$lib/components/events/EventTicket.svelte';
+
+	let selectedButton = 'layout'; // Default selected button
```

Comment:
@IgnacioDebat I believe naming here is a bit confusing!

I believe that what you want to create here is a toggle to see the events as a list or as a grid.

A much better naming would be something like:
- let viewEventsMode: 'grid' | 'list' = 'grid'

Reply:
And I believe you confused 'layout' with 'grid' 😅